install.sh

```diff
@@ -19,46 +19,46 @@ touch $HOME/.ssh/known_hosts
 
 echo "================= Installing basic packages ==================="
 apt-get install -y \
-  sudo  \
-  build-essential \
-  curl \
-  gcc \
-  make \
-  openssl \
-  software-properties-common \
-  wget \
-  nano \
-  unzip \
-  openssh-client \
-  libxslt-dev \
-  libxml2-dev \
-  htop \
-  gettext \
-  texinfo
+  sudo=1.8.9p5-1ubuntu1  \
+  build-essential=11.6ubuntu6 \
+  curl=7.35.0-1ubuntu2.10 \
+  gcc=4:4.8.2-1ubuntu6 \
+  make=3.81-8.2ubuntu3 \
+  openssl=1.0.1f-1ubuntu2.22 \
+  software-properties-common=0.92.37.8 \
+  wget=1.15-1ubuntu1.14.04.2 \
+  nano=2.2.6-1ubuntu1 \
+  unzip=6.0-9ubuntu1.5 \
+  openssh-client=1:6.6p1-2ubuntu2.8 \
+  libxslt1-dev=1.1.28-2ubuntu0.1 \
```

Comment:
This was renamed from `libxslt-dev` to `libxslt1-dev` as `libxslt-dev` actually installs `libxslt1-dev`.
```
root@59734d699a99:/# sudo apt-get install libxslt-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'libxslt1-dev' instead of 'libxslt-dev'
libxslt1-dev is already the newest version.
0 upgraded, 0 newly installed, 0 to remove and 8 not upgraded.
```

Also, the version was changed from `1.1.28-2build1` had conflicts.